inc/layers.php

```diff
@@ -122,25 +123,29 @@ function post_layers_box($post = false) {
       <table class="layers-list available-layers">
        <tbody data-bind="foreach: filteredLayers">
         <tr>
-         <td><strong data-bind="text: title"></strong></td>
+         <td><strong><a data-bind="text: title, attr: {href: url}" target="_new"></a></strong></td>
```

Comment:
Well, either one is behave the same for the browsers. Anyways, we should use "_blank" instead.